src/components/sections/heading.js

```diff
@@ -8,8 +8,7 @@ export const HeadingSection = ({ children, maxWidth, wrap = true }) => {
       sx={{
         textAlign: "center",
         mb: 2,
-        pb: 2,
-        whiteSpace: !wrap ? "nowrap" : "normal",
+        pb: 4,
```

Comment:
since the `wrap` variable isn't being used in this component, we can remove this from line 4 as well!